pkgs/development/python-modules/aw-client/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, poetry-core
+, aw-core
+, requests
+, persist-queue
+, click
+, tabulate
+, pytest
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "aw-client";
+  version = "0.5.11";
+
+  format = "pyproject";
+
+  # pypi distribution doesn't include tests, so build from source instead
+  src = fetchFromGitHub {
+    owner = "ActivityWatch";
+    repo = "aw-client";
+    rev = "v${version}";
+    sha256 = "sha256-5WKGRoZGY+QnnB1Jzlju5OmCJreYMD8am2kW3Wcjhlw=";
+  };
+
+  disabled = pythonOlder "3.8";
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    aw-core
+    requests
+    persist-queue
+    click
+    tabulate
+    typing-extensions
+  ];
+
+  nativeCheckInputs = [ pytest ];
+  checkPhase = ''
+    runHook preCheck
+
+    # Fake home folder for tests that write to $HOME
+    export HOME="$TMP/fake-test-home"
+    pytest tests/test_requestqueue.py
```

Comment:
use pytestCheckHook

Reply:
Should still keep the `pythonImportsCheck = [ "aw_client" ];`?

Reply:
Yes, please